src/components/augmentedReality/AugmentedReality.js

```diff
@@ -14,20 +15,26 @@ import { ARModal } from './ARModal';
 import { ARObjectList } from './ARObjectList';
 import { WhatIsARButton } from './WhatIsARButton';
 
-export const AugmentedReality = ({ navigation, onSettingsScreen, tourID }) => {
-  const {
-    data: staticData,
-    error,
-    loading,
-    refetch
-  } = useStaticContent({
-    name: `arDownloadableDataList-${tourID}`,
-    type: 'json'
-  });
+const OnSettingsScreen = ({ id, isLoading, setData }) => {
```

Comment:
we name components with `...Screen` if the present a screen, like index, detail ...
calling `return <OnSettingsScreen {...{ id, isLoading, setData }} />;` now suggests some screen component being rendered instead of a small component like this.
i suggest a rollback and working with `skip` in the `useQuery` to only call it on the settings screen.
feel free to ask again, if something is not clear.

Reply:
You're absolutely right. I think it's much clearer now.
https://github.com/ikuseiGmbH/smart-village-app-app/pull/608/commits/39ed66226c960aa10e02b3e40ebcb1e5c5432701